earthfile2llb/interpreter.go

```diff
@@ -268,11 +268,15 @@ func (i *Interpreter) handleRun(ctx context.Context, cmd spec.Command) error {
 			return Errorf(cmd.SourceLocation, "secrets need to be implemented for the LOCALLY directive")
 		}
 
-		err = i.converter.RunLocal(ctx, fs.Args(), *pushFlag)
-		if err != nil {
-			return WrapError(err, cmd.SourceLocation, "apply RUN")
+		if i.withDocker == nil {
```

Comment:
Nit: It's easier to read if this condition is reversed - the more rare cases being nested deeper.

Reply:
I have a follow up PR which will expand on handling the withDocker logic which is I suspect will be more complicated than the without logic. I'll swap it for now and might swap back if it helps with readability later.